packages/lib-classifier/src/store/SubjectStore.spec.js

```diff
@@ -68,78 +84,58 @@ describe('Model > SubjectStore', function () {
       })
     })
 
-    describe('after emptying the queue', function () {
+    describe('when the queue is not refilled', function () {
       before(function () {
-        sinon.stub(clientStub.panoptes, 'get').callsFake(() => Promise.resolve({ body: [] }))
-        while (rootStore.subjects.resources.size > 0) {
-          rootStore.subjects.advance()
-        }
+        sinon.stub(clientStub.panoptes, 'get').callsFake(() => Promise.resolve({ body: { subjects: [] } }))
       })
 
       after(function () {
         clientStub.panoptes.get.restore()
       })
 
-      it('should leave the active subject empty', function () {
+      it('should advance until empty without crashing', function () {
+        expect(rootStore.subjects.resources.size).to.equal(10)
+        while (rootStore.subjects.resources.size > 0) {
+          rootStore.subjects.advance()
+        }
+        expect(rootStore.subjects.resources.size).to.equal(0)
+        expect(rootStore.subjects.active).to.be.undefined()
```

Comment:
Should the active subject be undefined here, or should it be the last subject that was in the initial queue?